src/main/java/com/gmail/val59000mc/configuration/MainConfiguration.java

```diff
@@ -424,12 +433,22 @@ public void load(YamlFile cfg, @Nullable YamlFile storage){
 
 		// custom events
 		enableTimeEvent = cfg.getBoolean("custom-events.time.enable",false);
+		isEconomyTime = cfg.getBoolean("custom-events.time.isEconomy", false);
 		rewardTimeEvent = cfg.getDouble("custom-events.time.reward",0);
+		timeCommands = new ArrayList<>(cfg.getStringList("custom-events.time.commands"));
 		intervalTimeEvent = cfg.getLong("custom-events.time.interval",600);
+
 		enableKillEvent = cfg.getBoolean("custom-events.kill.enable",false);
+		isEconomyKill = cfg.getBoolean("custom-events.kill.isEconomy", false);
 		rewardKillEvent = cfg.getDouble("custom-events.kill.reward");
+		killCommands = new ArrayList<>(cfg.getStringList("custom-events.kill.commands"));
+
+
 		enableWinEvent = cfg.getBoolean("custom-events.win.enable",false);
+		isEconomyWin = cfg.getBoolean("custom-events.win.isEconomy", false);
 		rewardWinEnvent = cfg.getDouble("custom-events.win.reward",0);
+		winCommands = new ArrayList<>(cfg.getStringList("custom-events.win.commands"));
+
```

Comment:
Maybe loop trough the commands here to remove the initial / instead of redoing it every time the commands get executed.